kuma/wiki/management/commands/delete_old_spam_attempt_data.py

```diff
@@ -0,0 +1,18 @@
+"""
+Delete old DocumentSpamAttempt data
+"""
+from optparse import make_option
+
+from django.core.management.base import BaseCommand
+from kuma.wiki.tasks import delete_old_documentspamattempt_data
+
+
+class Command(BaseCommand):
+    help = "Delete old DocumentSpamAttempt.data"
+    option_list = BaseCommand.option_list + (
+        make_option('--days', dest="days", default=30, type=int,
+                    help="How many days 'old'"),
```

Comment:
Sorry, I meant django-constance. configman is something else entirely. :) You're closer to this than I am but I thought I'd ask.